src/components/NurseryWithdrawals/PlantingProgressList.tsx

```diff
@@ -91,24 +95,46 @@ export default function PlantingProgressList({
   const data = useAppSelector((state: any) => searchPlantingProgress(state, search.trim(), plantingCompleted));
   const [selectedRows, setSelectedRows] = useState<any[]>([]);
   const dispatch = useAppDispatch();
+  const defaultTimeZone = useDefaultTimeZone();
   const [requestId, setRequestId] = useState<string>('');
+  const [selectedZoneIdsBySiteId, setSelectedZoneIdsBySiteId] = useState<Record<number, number[]>>();
   const updatePlantingResult = useAppSelector((state) => selectUpdatePlantingsCompleted(state, requestId));
+  const subzonesStatisticsResult = useAppSelector((state) =>
+    selectZonesHaveStatistics(state, selectedZoneIdsBySiteId, defaultTimeZone.get().id)
+  );
   const snackbar = useSnackbar();
   const [showWarningModal, setShowWarningModal] = useState(false);
+  const { selectedOrganization } = useOrganization();
 
   useEffect(() => {
     if (data && hasZones === undefined) {
       setHasZones(data.some((d) => d.subzoneName));
     }
   }, [data, hasZones]);
 
+  useEffect(() => {
+    if (selectedRows) {
+      const zoneIds = selectedRows.reduce((selectedZoneIdsBySiteIdObj: Record<number, number[]>, row) => {
+        const siteId = row.siteId;
+        if (selectedZoneIdsBySiteIdObj[siteId]) {
+          selectedZoneIdsBySiteIdObj[siteId].push(row.zoneId);
```

Comment:
missed this.. we will have a list with duplicate zoneIds if multiple subzones belong to the same zone.
make this a set.. otherwise we increase the cost of the selector function